_component_design/cards.md

```diff
@@ -17,82 +17,53 @@ helix-ui-javascript: false
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-Use cards to present minimal high level information that allows the user to dig deeper. Do not use cards in place of hyper-robust data tables; also, be wary of using cards with a large quantity of data entries.
+## When to use
 
-### Best practices
-
-- Cards should be uniform in size.
-- Don’t overcrowd your card. It is not meant to hold a wealth of information, but high level views to dig deeper into.
-- Cards are not always the right solution to displaying information. If the information would be better presented in a list or table, do so, or present both a list or table and a [grid](#grids) options.
+Use cards when you want a visually distinct arrangement of tabular data, and especially when an informational summary of different data objects is helpful for the user.
 
 {% endcolumn %}
 
-{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
-{% figure [class:"image bg-light border"] %}
- <embed src="{{site.url}}/assets/images/components/content-areas/cards/cards-hero.png" width="600"/>
-{% endfigure %}
-{% endcolumn %}
-
 </div>
 
 </section>
 
-## Overview
-
 <section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-### Default layout
+## Best practices
 
-The card pattern is composed of the following three elements:
-
-- **Header (optional):** The header sets the context for a card component and allows action on the card through the application of icons to it.
-- **Body (required):** The body contains the primary content associated with the card. The body section details the underlying object, of which the card is representing to the user.
-- **Footer (optional):** The footer serves to provide users with the calls to action that a user need take upon a card when applicable.
+Use the following best practices when you construct cards:
 
+- Construct cards so they are uniform in size.
+- Do not use cards in place of robust data tables or with a large quantity of data entries.
+- The number of cards that you can use is unlimited. However, ensure that the page use case justifies the data.
+- Do not overuse cards in an application. Too many pages with cards might confuse users because they cannot intuitively determine the page that they are on.
+- Do not include too much information on a card. Include only high-level information on a card, and link to more detailed information.
+- If the information is better presented in a list or table, do not use cards. If you believe that card grids might appeal to some users, you can present both layout options.
```

Comment:
remove the belief language e.g.
If the option to view a tabular data as a card grid is useful, you can present both layout options.